### ISSUE TITLE
resolve aliases?

### ISSUE THREAD

**Issue by jrochkind** [cmalven/vite-plugin-sass-glob-import#7]
Thanks for this sweet plugin! (although the fact that it isn't more popular makes me wonder why others don't want it and if there's something I'm missing!)

I am new to vite, so I may not be understanding what's going on, but it's looking to me like it may somehow not be useable with vite resolves alises?

For instance, if I have in my vite.config:

```javascript
  resolve: {
    alias: {
      "@stylesheets": resolve(__dirname, 'app/frontend/stylesheets')
    }
  }
```

I can now use in a sass stylesheet, something like:

   @import "@stylesheets/special_files/one.scss";

this is a vite features, and it work for sass too, OK, great.  But with vite-plugin-sass-glob-import, should I be able to do:

    @import "@stylesheets/special_files/**/*.scss";

? I _think_ it does not work. 

It does not product any error -- trying to import a glob path whose prefix doesn't in fact exist seems to never produce an error, just silently not import anything?  Which is actually a little bit of a gotcha. 

But it does not seem to import what I expect. Am I doing something wrong, or is it expected that vite resolve aliases won't work here?

**Comment by cmalven** [comment]
@jrochkind Thanks for pointing this out. You're not doing anything wrong - this definitely won't work currently because whenever you use a glob import pattern this plugin takes over the importing and inlines the resulting files, so the path would never be parsed/resolved by Vite's default pipeline, and therefore the alias won't be resolved.

It is probably possible to support this, but it would require a quite a bit of work, so there's a good chance I won't get to it any time soon. I would gladly welcome a PR for it!

**Comment by jrochkind** [comment]
Cool, makes sense, thanks for the response. 

Not quite the same issue, but if I do a glob import `@import "prefix/that/doesn't/exist/*.scss`, I would find it more convenient if I got an error, instead of just a silent no-op. Normally, of course, if I do an import of a file that doesn't, I get an error. And I don't expect an error if the wildcard just doesn't match anything in a prefix that does exist... but when the prefix itself does not exist, it's easy to mis a typo or a misconfiguration (as above, when `@alias` wasn't resolved, it just turned into a no-op, rather than error like it would ordinarily be without a glob). 

What do you think?

**Comment by cmalven** [comment]
[1.5.0](https://github.com/cmalven/vite-plugin-sass-glob-import/releases/tag/1.5.0) will display a console warning when a glob pattern doesn't match existing directories. I opted for this instead of throwing a full error because I frequently intentionally include glob import patterns that I may or may not actually end up adding any files to, but still want to make it clear in the code where these files would be added. Example: `vendor-customization/**/*.scss`

**Comment by jrochkind** [comment]
Thanks! 

In your example, does even `vendor-customization/` not exist yet though?

What I was hoping for was an error only if the "prefix", the part before any `**` or components including `*`, didn't exist. Which is a bit different  thing than "the glob doesn't match any contents".  But perhaps that would be difficult to implement, or perhaps there's a use case for even the prefix, in that case `vendor-customization/` not existing and still not wanting errors.

**Comment by cmalven** [comment]
Yes, in my example I don't even add a `vendor-customization` directory until I actually need it, but the glob import is still in the code. So I think my use-case is the use-case you're describing. I can totally understand where you're coming from, too, but this seemed like a decent middle-ground where it is at least giving you a clue about why things might not be working while also allowing some flexibility.

**Comment by jrochkind** [comment]
OK, thanks for the clarification!

**Comment by cmalven** [comment]
@jrochkind I can't say what Ruby Sass did way back in the day, but I can say what the most popular sass glob plugin for Gulp does (https://www.npmjs.com/package/gulp-sass-glob, still at 23k weekly uses) because this plugin is based extremely heavily on it. It doesn't throw any kind of error or message at all for non-matching glob patterns - it silently ignores them just like this plugin did before 1.5.0

**Comment by jrochkind** [comment]
right on, thanks!
